src/types/dtypes.ts

```diff
@@ -204,6 +214,11 @@ export class DType {
     return this.value.arrayValue;
   }
 
+  /**
+   * Attempts to extract a arrayValue from a DType through an
+   * intermediary conversion (first tried is array, then double)
+   * @param value
```

Comment:
I agree, but it looked like it was boilerplate code that wasn't going to need changing so I thought it would be worth it even just so I can see the function description when I hover over the function in other files.